index.js

```diff
@@ -163,4 +178,30 @@ PagedContacts.authorized = PagedContactsModule.authorized;
 if (Platform.OS === 'ios') {
   PagedContacts.notDetermined = PagedContactsModule.notDetermined;
   PagedContacts.restricted = PagedContactsModule.restricted;
-}
\ No newline at end of file
+}
+
+/**
+ * The `PagedContactsImage` class is a simple wrapper around the `Image` component
+ * using `PagedContacts.setImageViewWithHandle`
+ * function to update the view with the contact picture on the native side.
+ * 
+ * @prop {Number} contactId Contact identifier
+ * @prop {String} imageType Image type - either `image` or `thumbnailImage`
+ * 
+ * @memberOf PagedContacts
+ */
+class PagedContactsImage extends Component {
```

Comment:
Good, so for a quick fix for One App, we don't have to push this in master. Create a branch, create a custom tag for One App on npm and use it from there. This is not a scalable solution. The proper solution is looking into the bridge and why it is slow. 

Regarding iOS thumbnails, that's irrelevant because there is no performance issue.